megamek/src/megamek/common/logging/LogLevel.java

```diff
@@ -49,6 +49,10 @@ public int toInt() {
         return level.toInt();
     }
 
+    public boolean willLog(final LogLevel logLevel) {

```

Comment:
This method is much better than what you were doing before. I like it. Nice work.